cfme/v2v/migrations.py

```diff
@@ -664,6 +679,11 @@ class InfrastructureMapping(BaseEntity):
     description = attr.ib(default=None)
     form_data = attr.ib(default=None)
 
+    def update(self, form_data):
```

Comment:
**Question**: Why do you think this should be in Entity while Create and Delete were in Collection? We may need to consider all the operations and their placement.

Reply:
This should be in entity as update is for a single mapping . generally we add delete too in Baseentity and Methods like create in collection .